docs/content/getting-started/docker.md

```diff
@@ -48,3 +48,17 @@ For example, for running cardano-node + ogmios on the testnet, listening to tcp/
 ```console
 $ NETWORK=testnet OGMIOS_PORT=1338 docker-compose --project-name cardano-ogmios-testnet up
 ```
+
+## As a single image
+
+If you're not fond of docker-compose and the container orchestration it requires, you may also build (or pull) an image to run both a `cardano-node` and `ogmios` side-by-side, within the same container. Note that this is the default build target when building ogmios with Docker, but we can be explicit when building the image using `--target cardano-node-ogmios`. For example:
+
+```console
+$ docker build --target cardano-node-ogmios --tag ktorz/cardano-node-ogmios:latest . 
+```
+
+Images with either release tags or `:latest` tags are also uploaded to the corresponding [DockerHub registry](https://hub.docker.com/r/ktorz/cardano-node-ogmios) and you may also pull them directly from there. Assuming you've pulled `ktorz/cardano-node-ogmios:latest`, you can start a container by running:
+
+```console
+$ docker run --detach --name cardano-node-ogmios --e NETWORK testnet -p 1337:1337 ktorz/cardano-node-ogmios:latest
```

Comment:
I've been using `1338` for `testnet`. Do you think it's worth using this convention to help with support and education?

```
-p 1338:1337
```

Reply:
Also, in quick start guides I feel it's better to provide the run command without specifying detached mode, as the goal is to just see it working. Another option is to add `&& docker logs -f  cardano-node-ogmios`, but I prefer the former since anyone needing the process to be detached probably knows what to do.

Reply:
> I've been using 1338 for testnet. Do you think it's worth using this convention to help with support and education?

And I've been wondering why :sweat_smile: ? I see little chance for people running both mainnet and testnet at the same time, and network discriminant in addresses will already provide a good mechanism for silly mistakes. 

I fail to see the rationale here :no_mouth:  ?

---

1. :heavy_check_mark: 
2. :heavy_check_mark:

Reply:
> I fail to see the rationale here no_mouth ?

If you bind `1337` to a testnet container, then you have to destroy it before running a `mainnet` container on the same host. Not a big deal, so happy to stick with `1337`